riak_repl/datadog_checks/riak_repl/riak_repl.py

```diff
@@ -76,30 +83,58 @@ def check(self, instance):
         except ValueError:
             raise CheckException('{} returned an unserializable payload'.format(url))
 
+        cluster = stats['cluster_name']
+
         for key, val in iteritems(stats):
             if key in self.REPL_STATS:
-                self.safe_submit_metric("riak_repl." + key, val, tags=tags)
+                self.safe_submit_metric("riak_repl." + key, val, tags=tags + ['cluster:%s' % cluster])
 
-        if stats['realtime_enabled'] is not None:
+        if stats['realtime_started'] is not None:
             for key, val in iteritems(stats['realtime_queue_stats']):
                 if key in self.REALTIME_QUEUE_STATS:
-                    self.safe_submit_metric("riak_repl.realtime_queue_stats." + key, val, tags=tags)
-
-        for c in stats['connected_clusters']:
-            cluster = c.replace("-", "_")
-            if c not in stats['fullsync_coordinator']:
-                continue
-            for key, val in iteritems(stats['fullsync_coordinator'][c]):
-                if key in self.FULLSYNC_COORDINATOR:
-                    self.safe_submit_metric("riak_repl.fullsync_coordinator." + cluster + "." + key, val, tags=tags)
+                    self.safe_submit_metric(
+                        "riak_repl.realtime_queue_stats." + key, val, tags=tags + ['cluster:%s' % cluster]
+                    )
+
+        for c in connected_clusters:
+
+            if stats['fullsync_enabled'] is not None:
+                if self.exists(stats['fullsync_coordinator'], [c]):
+                    for key, val in iteritems(stats['fullsync_coordinator'][c]):
+                        if key in self.FULLSYNC_COORDINATOR:
+                            self.safe_submit_metric(
+                                "riak_repl.fullsync_coordinator." + key, val, tags=tags + ['cluster:%s' % c]
```

Comment:
Just two notes:
* People upgrading need to defined connected_clusters if they want to continue getting `riak_repl.fullsync_coordinator....` metrics
* Before the metric names where `"riak_repl.fullsync_coordinator." + cluster + "." + key` and now is `riak_repl.fullsync_coordinator." + key` with a cluster tag

So I think this should be a major version bump because it has breaking changes

Reply:
This makes sense. bumped